<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="dns-prefetch" href="//www.google.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.07.22.21260952v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.07.22.21260952v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.07.22.21260952v2.ppt" />
<meta name="article_thumbnail" content="https://www.medrxiv.org/content/medrxiv/early/2021/10/22/2021.07.22.21260952/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/medrxiv/early/2021/10/22/2021.07.22.21260952.atom" />
<meta name="HW.pisa" content="medrxiv;2021.07.22.21260952v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations" />
<meta name="DC.Identifier" content="10.1101/2021.07.22.21260952" />
<meta name="DC.Date" content="2021-10-22" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press" />
<meta name="DC.Rights" content="© 2021, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Never before such a vast amount of data, including genome sequencing, has been collected for any viral pandemic than for the current case of COVID-19. This offers the possibility to trace the virus evolution and to assess the role mutations play in its spread within the population, in real time. To this end, we focused on the Spike protein for its central role in mediating viral outbreak and replication in host cells. Employing the Levenshtein distance on the Spike protein sequences, we designed a machine learning algorithm yielding a temporal clustering of the available dataset. From this, we were able to identify and define emerging persistent variants that are in agreement with known evidences. Our novel algorithm allowed us to define persistent variants as chains that remain stable over time and to highlight emerging variants of epidemiological interest as branching events that occur over time. Hence, we determined the relationship and temporal connection between variants of interest and the ensuing passage to dominance of the current variants of concern. Remarkably, the analysis and the relevant tools introduced in our work serve as an early warning for the emergence of new persistent variants once the associated cluster reaches 1% of the time-binned sequence data. We validated our approach and its effectiveness on the onset of the Alpha variant of concern. We further predict that the recently identified lineage AY.4.2 (‘Delta plus’) is causing a new emerging variant. Comparing our findings with the epidemiological data we demonstrated that each new wave is dominated by a new emerging variant, thus confirming the hypothesis of the existence of a strong correlation between the birth of variants and the pandemic multi-wave temporal pattern. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.

Highlights 

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

None

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Not applicable.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All raw data used in this work are obtained from open-source repositories: &lt;https://www.gisaid.org&gt; for the sequencing and &lt;https://ourworldindata.org/&gt; for the epidemiological data. The Machine Learning code is available at &lt;https://github.com/AdeledeHoffer/ML-Covid&gt;

&lt;https://github.com/AdeledeHoffer/ML-Covid&gt;" />
<meta name="DC.Contributor" content="Adele de Hoffer" />
<meta name="DC.Contributor" content="Shahram Vatani" />
<meta name="DC.Contributor" content="Corentin Cot" />
<meta name="DC.Contributor" content="Giacomo Cacciapaglia" />
<meta name="DC.Contributor" content="Maria Luisa Chiusano" />
<meta name="DC.Contributor" content="Andrea Cimarelli" />
<meta name="DC.Contributor" content="Francesco Conventi" />
<meta name="DC.Contributor" content="Antonio Giannini" />
<meta name="DC.Contributor" content="Stefan Hohenegger" />
<meta name="DC.Contributor" content="Francesco Sannino" />
<meta name="article:published_time" content="2021-10-22" />
<meta name="citation_title" content="Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Never before such a vast amount of data, including genome sequencing, has been collected for any viral pandemic than for the current case of COVID-19. This offers the possibility to trace the virus evolution and to assess the role mutations play in its spread within the population, in real time. To this end, we focused on the Spike protein for its central role in mediating viral outbreak and replication in host cells. Employing the Levenshtein distance on the Spike protein sequences, we designed a machine learning algorithm yielding a temporal clustering of the available dataset. From this, we were able to identify and define emerging persistent variants that are in agreement with known evidences. Our novel algorithm allowed us to define persistent variants as chains that remain stable over time and to highlight emerging variants of epidemiological interest as branching events that occur over time. Hence, we determined the relationship and temporal connection between variants of interest and the ensuing passage to dominance of the current variants of concern. Remarkably, the analysis and the relevant tools introduced in our work serve as an early warning for the emergence of new persistent variants once the associated cluster reaches 1% of the time-binned sequence data. We validated our approach and its effectiveness on the onset of the Alpha variant of concern. We further predict that the recently identified lineage AY.4.2 (`Delta plus`) is causing a new emerging variant. Comparing our findings with the epidemiological data we demonstrated that each new wave is dominated by a new emerging variant, thus confirming the hypothesis of the existence of a strong correlation between the birth of variants and the pandemic multi-wave temporal pattern. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.&lt;/p&gt;" />
<meta name="citation_journal_title" content="medRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press" />
<meta name="citation_publication_date" content="2021/01/01" />
<meta name="citation_mjid" content="medrxiv;2021.07.22.21260952v2" />
<meta name="citation_id" content="2021.07.22.21260952v2" />
<meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" />
<meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2.abstract" />
<meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2.full" />
<meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2021/10/22/2021.07.22.21260952.full.pdf" />
<meta name="citation_doi" content="10.1101/2021.07.22.21260952" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_firstpage" content="2021.07.22.21260952" />
<meta name="citation_author" content="Adele de Hoffer" />
<meta name="citation_author_institution" content="Politecnico di Torino" />
<meta name="citation_author" content="Shahram Vatani" />
<meta name="citation_author_institution" content="Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822" />
<meta name="citation_author_institution" content="Université de Lyon, Université Claude Bernard Lyon 1" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1277-5829" />
<meta name="citation_author" content="Corentin Cot" />
<meta name="citation_author_institution" content="Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822" />
<meta name="citation_author_institution" content="Université de Lyon, Université Claude Bernard Lyon 1" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5845-6500" />
<meta name="citation_author" content="Giacomo Cacciapaglia" />
<meta name="citation_author_institution" content="Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822" />
<meta name="citation_author_institution" content="Université de Lyon, Université Claude Bernard Lyon 1" />
<meta name="citation_author_email" content="g.cacciapaglia@ipnl.in2p3.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3426-1618" />
<meta name="citation_author" content="Maria Luisa Chiusano" />
<meta name="citation_author_institution" content="Department of Agricultural Sciences, Università degli Studi di Napoli Federico II" />
<meta name="citation_author_institution" content="Department of Research Infrastructures for Marine Biological Resources (RIMAR)" />
<meta name="citation_author" content="Andrea Cimarelli" />
<meta name="citation_author_institution" content="Centre International de Recherche en Infectiologie (CIRI), Univ Lyon, Inserm, U1111, Université Claude Bernard Lyon 1, CNRS, UMR5308" />
<meta name="citation_author" content="Francesco Conventi" />
<meta name="citation_author_institution" content="INFN sezione di Napoli, Complesso Universitario di Monte S. Angelo Edificio 6" />
<meta name="citation_author_institution" content="Dipartimento di Ingegneria Università degli studi di Napoli Parthenope, Centro Direzionale di Napoli" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5575-1413" />
<meta name="citation_author" content="Antonio Giannini" />
<meta name="citation_author_institution" content="University of Science and Technology of China (USTC)" />
<meta name="citation_author" content="Stefan Hohenegger" />
<meta name="citation_author_institution" content="Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822" />
<meta name="citation_author_institution" content="Université de Lyon, Université Claude Bernard Lyon 1" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6564-0795" />
<meta name="citation_author" content="Francesco Sannino" />
<meta name="citation_author_institution" content="INFN sezione di Napoli, Complesso Universitario di Monte S. Angelo Edificio 6" />
<meta name="citation_author_institution" content="Dipartimento di Fisica E. Pancini, Università di Napoli Federico II, Complesso Universitario di Monte S. Angelo Edificio 6" />
<meta name="citation_author_institution" content="Scuola Superiore Meridionale" />
<meta name="citation_author_institution" content="CP3-Origins &amp; the Danish Institute for Advanced Study, University of Southern Denmark" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2361-5326" />
<meta name="citation_reference" content="Taubenberger, J. K. &amp; Morens, D. M. 1918 influenza: The mother of all pandemics. Rev Biomed 17(1), 69–79 (2006)." />
<meta name="citation_reference" content="Sanjuán, R., Nebot, M. R., Chirico, N., Mansky, L. M. &amp; Belshaw, R. Viral mutation rates. J. Virol. 84, 9733–9748 (2010). DOI https://doi.org/10.1128/JVI.00694-10. https://jvi.asm.org/content/84/19/9733.full.pdf." />
<meta name="citation_reference" content="Plante, J. A. et al. Spike mutation d614g alters sars-cov-2 fitness. Nature 592, 116 –121 (2021). DOI https://doi.org/10.1038/s41586-020-2895-3." />
<meta name="citation_reference" content="Korber, B. et al. Tracking changes in sars-cov-2 spike: Evidence that d614g increases infectivity of the covid-19 virus. Cell 182, 812–827.e19 (2020). DOI https://doi.org/10.1016/j.cell.2020.06.043. https://www.sciencedirect.com/science/article/pii/S0092867420308205." />
<meta name="citation_reference" content="Wu, A. et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe 27(3), 325–328 (2020). DOI https://doi.org/10.1016/j.chom.2020.02.001." />
<meta name="citation_reference" content="Konings, F. et al. Sars-cov-2 variants of interest and concern naming scheme conducive for global discourse. Nat. Microbioliogy 6, 821–823 (2021). DOI https://doi.org/10.1038/s41564-021-00932-w." />
<meta name="citation_reference" content="Rambaut, A. et al. A dynamic nomenclature proposal for sars-cov-2 lineages to assist genomic epidemiology. Nat. Microbiol. 5, 1403–1407 (2020). DOI https://doi.org/10.1038/s41564-020-0770-5." />
<meta name="citation_reference" content="Elbe, S. &amp; Buckland-Merret, G. Data, disease and diplomacy: Gisaid’s innovative contribution to global health. Glob. Challenges 1, 33–46 (2017). DOI https://doi.org/10.1002/gch2.1018." />
<meta name="citation_reference" content="Shu, Y. &amp; McCauley, J. Gisaid: Global initiative on sharing all influenza data – from vision to reality. EuroSurveillance 22 (13) (2017). DOI https://doi.org/10.2807/1560-7917.ES.2017.22.13.30494." />
<meta name="citation_reference" content="Rambaud, A. et al. Preliminary genomic characterisation of an emergent sars-cov-2 lineage in the uk defined by a novel set of spike mutations. COVID-19 Genomics Consortium UK (CoG-UK) Rep. (2020). URL https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined563." />
<meta name="citation_reference" content="Mahase, E. Covid-19: What have we learnt about the new variant in the uk? BMJ 371 (2020). DOI https://doi.org/10.1136/bmj.m4944. https://www.bmj.com/content/371/bmj.m4944.full.pdf." />
<meta name="citation_reference" content="Tegally, H. et al. Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (sars-cov-2) lineage with multiple spike mutations in south africa. medRxiv (2020). DOI https://doi.org/10.1101/2020.12.21.20248640. https://www.medrxiv.org/content/early/2020/12/22/2020.12.21.20248640.full.pdf." />
<meta name="citation_reference" content="Sabino, E. C. et al. Resurgence of covid-19 in manaus, brazil, despite high seroprevalence. The Lancet 397, 452–455 (2021). DOI https://doi.org/10.1016/S0140-6736(21)00183-5." />
<meta name="citation_reference" content="Pater, A. A. et al. Emergence and evolution of a prevalent new sars-cov-2 variant in the united states. bioRxiv (2021). DOI https://doi.org/10.1101/2021.01.11.426287. https://www.biorxiv.org/content/early/2021/01/19/2021.01.11.426287.full.pdf." />
<meta name="citation_reference" content="Rasigade, J.-P. et al. A viral perspective on worldwide non-pharmaceutical interventions against covid-19. medRxiv (2020). DOI https://doi.org/10.1101/2020.08.24.20180927. https://www.medrxiv.org/content/early/2020/10/31/2020.08.24.20180927.full.pdf." />
<meta name="citation_reference" content="Volz, E. et al. Transmission of sars-cov-2 lineage b.1.1.7 in england: Insights from linking epidemiological and genetic data. medRxiv (2021). DOI https://doi.org/10.1101/2020.12.30.20249034. https://www.medrxiv.org/content/early/2021/01/04/2020.12.30.20249034.1.full.pdf." />
<meta name="citation_reference" content="Kermack, W. O., McKendrick, A. &amp; Walker, G. T. A contribution to the mathematical theory of epidemics. Proc. Royal Soc. A 115, 700–721 (1927). DOI https://doi.org/10.1098/rspa.1927.0118. https://royalsocietypublishing.org/doi/10.1098/rspa.1927.0118." />
<meta name="citation_reference" content="Perc, M. et al. Statistical physics of human cooperation. Phys. Reports 687, 1 –51 (2017). DOI https://doi.org/10.1016/j.physrep.2017.05.004. http://www.sciencedirect.com/science/article/pii/S0370157317301424." />
<meta name="citation_reference" content="Wang, Z., Andrews, M. A., Wu, Z.-X., Wang, L. &amp; Bauch, C. T. Coupled disease–behavior dynamics on complex networks: A review. Phys. Life Rev. 15, 1 –29 (2015). DOI https://doi.org/10.1016/j.plrev.2015.07.006. http://www.sciencedirect.com/science/article/pii/S1571064515001372." />
<meta name="citation_reference" content="Giordano, G. et al. Modeling vaccination rollouts, sars-cov-2 variants and the requirement for non-pharmaceutical interventions in italy. Nat. Medicine (2021). DOI https://doi.org/10.1038/s41591-021-01334-5." />
<meta name="citation_reference" content="Della Morte, M., Orlando, D. &amp; Sannino, F. Renormalization Group Approach to Pandemics: The COVID-19 Case. Front. Phys. 8, 144 (2020). DOI https://doi.org/10.3389/fphy.2020.00144." />
<meta name="citation_reference" content="Cacciapaglia, G. &amp; Sannino, F. Interplay of social distancing and border restrictions for pandemics (COVID-19) via the epidemic Renormalisation Group framework. Sci Rep 10, 15828 (2020). DOI https://doi.org/10.1038/s41598-020-72175-4.2005.04956." />
<meta name="citation_reference" content="Cacciapaglia, G., Cot, C. &amp; Sannino, F. Second wave covid-19 pandemics in europe: A temporal playbook. Sci Rep 10, 15514 (2020). DOI https://doi.org/10.1038/s41598-020-72611-5.2007.13100." />
<meta name="citation_reference" content="Cacciapaglia, G. et al. Epidemiological theory of virus variants (2021). 2106.14982." />
<meta name="citation_reference" content="Levenshtein, V. I. Binary codes capable of correcting deletions, insertions, and reversals. Doklady Akademii Nauk 163, 845–848 (1965)." />
<meta name="citation_reference" content="Levenshtein, V. I. Binary codes capable of correcting deletions, insertions, and reversals. Cybern. Control. Theory 10, 707–710 (1966)." />
<meta name="citation_reference" content="Wilson, K. G. Renormalization group and critical phenomena. 1. Renormalization group and the Kadanoff scaling picture. Phys. Rev. B 4, 3174–3183 (1971). DOI https://doi.org/10.1103/PhysRevB.4.3174." />
<meta name="citation_reference" content="Wilson, K. G. Renormalization group and critical phenomena. 2. Phase space cell analysis of critical behavior. Phys. Rev. B 4, 3184–3205 (1971). DOI https://doi.org/10.1103/PhysRevB.4.3184." />
<meta name="citation_reference" content="Cacciapaglia, G., Cot, C., Islind, A. S., Óskarsdóttir, M. &amp; Sannino, F. Impact of us vaccination strategy on covid-19 wave dynamics. Sci. Reports 11, 10960 (2021). DOI https://doi.org/10.1038/s41598-021-90539-2. 2012.12004." />
<meta name="citation_reference" content="Della Morte, M. &amp; Sannino, F. Renormalization group approach to pandemics as a time-dependent sir model. Front. Phys. 8 (2021). DOI https://doi/org/10.3389/fphy.2020.591876." />
<meta name="citation_reference" content="Cacciapaglia, G. et al. The field theoretical abc of epidemic dynamics (2021). 2101.11399." />
<meta name="citation_reference" content="Cacciapaglia, G., Cot, C. &amp; Sannino, F. Multiwave pandemic dynamics explained: How to tame the next wave of infectious diseases. Sci. Reports 11, 6638 (2021). DOI https://doi.org/10.1038/s41598-021-85875-2.2011.12846." />
<meta name="citation_reference" content="Cacciapaglia, G. &amp; Sannino, F. Evidence for complex fixed points in pandemic data. Front. Appl. Math. Stat. 7, 659580 (2021). DOI https://doi.org/10.3389/fams.2021.659580.2009.08861." />
<meta name="citation_reference" content="Latif, A. A. et al. AY.4.2 Lineage Report. outbreak.info (2021). URL https://outbreak.info/situation-reports?pango=AY.4.2." />
<meta name="citation_reference" content="Wrobel, A. G. et al. SARS-CoV-2 and bat RaTG13 spike glycoprotein structures inform on virus evolution and furincleavage effects. Nat Struc Mol Biol 27(8), 763–767 (2020). DOI https://doi.org/10.1038/s41594-020-0468-7." />
<meta name="citation_reference" content="Thomson, E. C. et al. Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity. Cell 184(4), 1171–1187 (2021). DOI https://doi.org/10.1016/j.cell.2021.01.037." />
<meta name="twitter:title" content="Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations" />
<meta name="twitter:site" content="@medrxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:description" content="Never before such a vast amount of data, including genome sequencing, has been collected for any viral pandemic than for the current case of COVID-19. This offers the possibility to trace the virus evolution and to assess the role mutations play in its spread within the population, in real time. To this end, we focused on the Spike protein for its central role in mediating viral outbreak and replication in host cells. Employing the Levenshtein distance on the Spike protein sequences, we designed a machine learning algorithm yielding a temporal clustering of the available dataset. From this, we were able to identify and define emerging persistent variants that are in agreement with known evidences. Our novel algorithm allowed us to define persistent variants as chains that remain stable over time and to highlight emerging variants of epidemiological interest as branching events that occur over time. Hence, we determined the relationship and temporal connection between variants of interest and the ensuing passage to dominance of the current variants of concern. Remarkably, the analysis and the relevant tools introduced in our work serve as an early warning for the emergence of new persistent variants once the associated cluster reaches 1% of the time-binned sequence data. We validated our approach and its effectiveness on the onset of the Alpha variant of concern. We further predict that the recently identified lineage AY.4.2 (‘Delta plus’) is causing a new emerging variant. Comparing our findings with the epidemiological data we demonstrated that each new wave is dominated by a new emerging variant, thus confirming the hypothesis of the existence of a strong correlation between the birth of variants and the pandemic multi-wave temporal pattern. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.

Highlights 

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

None

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Not applicable.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All raw data used in this work are obtained from open-source repositories: &lt;https://www.gisaid.org&gt; for the sequencing and &lt;https://ourworldindata.org/&gt; for the epidemiological data. The Machine Learning code is available at &lt;https://github.com/AdeledeHoffer/ML-Covid&gt;

&lt;https://github.com/AdeledeHoffer/ML-Covid&gt;" />
<meta name="og-title" property="og:title" content="Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations" />
<meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" />
<meta name="og-site-name" property="og:site_name" content="medRxiv" />
<meta name="og-description" property="og:description" content="Never before such a vast amount of data, including genome sequencing, has been collected for any viral pandemic than for the current case of COVID-19. This offers the possibility to trace the virus evolution and to assess the role mutations play in its spread within the population, in real time. To this end, we focused on the Spike protein for its central role in mediating viral outbreak and replication in host cells. Employing the Levenshtein distance on the Spike protein sequences, we designed a machine learning algorithm yielding a temporal clustering of the available dataset. From this, we were able to identify and define emerging persistent variants that are in agreement with known evidences. Our novel algorithm allowed us to define persistent variants as chains that remain stable over time and to highlight emerging variants of epidemiological interest as branching events that occur over time. Hence, we determined the relationship and temporal connection between variants of interest and the ensuing passage to dominance of the current variants of concern. Remarkably, the analysis and the relevant tools introduced in our work serve as an early warning for the emergence of new persistent variants once the associated cluster reaches 1% of the time-binned sequence data. We validated our approach and its effectiveness on the onset of the Alpha variant of concern. We further predict that the recently identified lineage AY.4.2 (‘Delta plus’) is causing a new emerging variant. Comparing our findings with the epidemiological data we demonstrated that each new wave is dominated by a new emerging variant, thus confirming the hypothesis of the existence of a strong correlation between the birth of variants and the pandemic multi-wave temporal pattern. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.

Highlights 

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

None

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Not applicable.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All raw data used in this work are obtained from open-source repositories: &lt;https://www.gisaid.org&gt; for the sequencing and &lt;https://ourworldindata.org/&gt; for the epidemiological data. The Machine Learning code is available at &lt;https://github.com/AdeledeHoffer/ML-Covid&gt;

&lt;https://github.com/AdeledeHoffer/ML-Covid&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="" />
<meta name="citation_date" content="2021-10-22" />
<meta name="description" content="medRxiv - The Preprint Server for Health Sciences" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" />
<link rel="shortlink" href="https://www.medrxiv.org/node/425607" />
    <title>Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations | medRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__DcKr3CumsZTU8f2X6BlpTbZkTl0a4ktIvtu7FpBmIVw__TTo2i8J-9814EgFI83jP9bGBs2EjZDgQa8pyzUSxjOU__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PyYHFwY5MxAn4BX77TYDdU4RDXeQS5jXeWU_EhtPpxE__Qw-rezBwY6q5lzhujTh7mg-q1yvjz_nglxTt9kKsaF0__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__XPg6Aobs217v0T9rA_Y-wMrH3jyqmqunImGwMN4WMEE__Nc8_6i0mjd_g-hzHvpxLmo5g2aluEkJf5SXcEKb7YRY__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__GJ_y0JYBjHihOcdJg-wLEQWRfqvBxU74PCdjUcUHlyY__f0nQteVSjWTKq2ga8E2qYttGognShmPZZJdeGhQoNFI__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all" />
    <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__cTu5xPrxXhjVN5aAnPJMuPpBY19xTkrTcv2IlQCNS94__DbBDIaPID44_lOD9MTlulOTk4IJuDRxONi_dobBuM1g__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.js"></script>
<script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&amp;render=explicit&amp;onload=drupalRecaptchaOnload"></script>
<script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__6hIesE0QCBXoXlEIn01f-78o39DmDNEoInQ35EHxoXw__qAjUcZEMaB7nsfhw64tJNfG5xuXddaTuPmbIfa6fPMk__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qQutIv12arcKdRSi0SL1hYL9ZhZQAUGf8nCsU5RnfW4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"425607","apath":"\/medrxiv\/early\/2021\/10\/22\/2021.07.22.21260952.atom","pisa":"medrxiv;2021.07.22.21260952v2","ac":{"\/medrxiv\/early\/2021\/10\/22\/2021.07.22.21260952.atom":{"access":{"full":true},"pisa_id":"","apath":"\/medrxiv\/early\/2021\/10\/22\/2021.07.22.21260952.atom","jcode":"medrxiv"}},"processed":["highwire_math"],"markup":[{"requested":"abstract","variant":"abstract","view":"abstract","pisa":"medrxiv;2021.07.22.21260952v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.07.22.21260952v2","title":"Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations","identifier":"node\/425607"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022217Ng7+4u1hQFednGAt3RGeGNP6kWzeu2noSKdftK983937A45182u3sJHLRp9gHRaH7NjzUR1hWv1009I\\\\\\\/PXJDM5HFlpeqCwT0QslRu1taaFWYg5oMZC2RLEhAN+R6rN\\\\\\\/BvfbKqO9\\\\\\\/9QLGmVIVFkTV9\\\\\\\/uV9iDh\\\\\\\/OgFeu4sZB2aqH50fJNxReFkV6GJd8pIcXLWhbWrmaPdyRaICd+bDY7yHaMtc5YfQIqI2m4fQyzQ8yIZVEtoISM3iT8of4arCu6dgM\\\\\\\/0Fw6ZwKk3HzSQmoJvnUxgqpflJKRTc2NdMuIf\\\\\\\/yhER5s6uXS7WDNw\\\\\\\/juq2eTKl9FIjD+xuBcpLxRTghZbgs7iXR8Eke9I6+7Tknp\\\\\\\/0TLvDmW6m98b5GJuPJJvObDcjshuXjUoGmlrVTlqNTwxkwg0ePfImllc0lsytMoF68TNXkUymkwDxIOE5C+jGQsGnn4eX99tYt7+VErv\\\\\\\/ayyqRewjyjkfH85xA6V70Xtckk7iROv4tcn\\\\\\\/S+j+FyvJMmvM2ENEJGWC3Xd4XZe4kMqTl2Owu4+qscH\\\\\\\/UtceuSzK8IhakO6G7mNWnL6GpANbjvEniuNYm9618YevWfZ+1iZqudQPl+Umnqb6jrM5tg\\\\\\\/DnYbzGM9MadRc1c8M99Ujv59N8X4ofQXyip647M4uNM\\\\\\\/aXy1ictAJCweV7UlIxycnhkIl9HJDTujR8ryNgTMWkggKie2yIzGiN3fGWU4Nm2+lIZTesoADpQEaS8Ictpi0Ax98UdxvD2UaPrNVRWYhM6ny9QQcBYby2hQ2p30IiZ\\\\\\\/oHnvLy7HG4KG16p6SvUWJS9RaHB3CYbG0pJkMybGBRD17aYVRqZ6SEmzehY8civ05Ah5fn4FZ2X6Vy\\\\\\\/TcXnntwYYEnHSER43zgjQFFsqW\\\\\\\/vitQwXAB1br2VuvlHfkmiETa46iF7xQQGqoHC3ue+ZjOtWY00KMGKeeMlWYVuXHhaKqH+UztckRMduYOBMPGbVeJUA7AA1uoX\\\\\\\/CA1BRo\\\\\\\/m5xhC6AQIX0apGin7aCSrHR76zIypObeam7oRe++Fy3C443Wkjb3\\\\\\\/YGGVvXtUwUZ8M9h3VD3rwtIRs7bFlos8\\\\\\\/YqC1TdlGSYStuqFFf+lPKhv1XtORsAOzsIZqnepPR5Y4pTjmNlGJFmv5Yb9WEuDmwVvdmm1joXxneVmiAzsESOSr7rNK2BzStLTouYQUJ\\\\\\\/sQ5uOWXFm8prB4tpkG7Ppxl58lbU3rNF6RXKJZIUl4ZnYt6uqpy0wKOKSKK5a1JxBmyut2zUQJ1CtCv8BCLFzq\\\u0022,\\\u0022iv\\\u0022:\\\u0022BR2pzs66Ndgnlopr4bXNSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022be2296841650a362d4fb067bc70978a22daec5585b6656f8d8103b91fe17e2f5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.07.22.21260952v2":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-425607 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.07.22.21260952v2" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><span class="icon-search"></span><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ws84SFjgk6heauhYdZB3LGcyykI8uyM53FLrKjYSmk8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node425607" data-node-nid="425607" id="node425607--2629452616" data-pisa="medrxiv;2021.07.22.21260952v2" data-pisa-master="medrxiv;2021.07.22.21260952" data-apath="/medrxiv/early/2021/10/22/2021.07.22.21260952.atom" data-hw-author-tooltip-instance=""><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip" >

  
      <h1 class="highwire-cite-title" id="page-title">Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Adele</span> <span class="nlm-surname">de Hoffer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1277-5829" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Shahram</span> <span class="nlm-surname">Vatani</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5845-6500" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Corentin</span> <span class="nlm-surname">Cot</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3426-1618" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Giacomo</span> <span class="nlm-surname">Cacciapaglia</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Maria Luisa</span> <span class="nlm-surname">Chiusano</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Cimarelli</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-5575-1413" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Conventi</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Antonio</span> <span class="nlm-surname">Giannini</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6564-0795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Hohenegger</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-2361-5326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Sannino</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2021.07.22.21260952 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node425607--2629452616" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adele de Hoffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Politecnico di Torino</span>, Corso Duca degli Abruzzi 24, 10129 Torino, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adele%2Bde%2BHoffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Hoffer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdele%2Bde%2BHoffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shahram Vatani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822</span>, 69622 Villeurbanne, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Université de Lyon, Université Claude Bernard Lyon 1</span>, 69001 Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shahram%2BVatani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vatani%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShahram%2BVatani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1277-5829" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shahram Vatani</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Corentin Cot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822</span>, 69622 Villeurbanne, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Université de Lyon, Université Claude Bernard Lyon 1</span>, 69001 Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corentin%2BCot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cot%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorentin%2BCot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5845-6500" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corentin Cot</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Giacomo Cacciapaglia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822</span>, 69622 Villeurbanne, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Université de Lyon, Université Claude Bernard Lyon 1</span>, 69001 Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giacomo%2BCacciapaglia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cacciapaglia%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGiacomo%2BCacciapaglia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3426-1618" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Giacomo Cacciapaglia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:g.cacciapaglia@ipnl.in2p3.fr" class="" data-icon-position="" data-hide-link-title="0">g.cacciapaglia@ipnl.in2p3.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria Luisa Chiusano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Agricultural Sciences, Università degli Studi di Napoli Federico II</span>, Portici, 80055 <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Research Infrastructures for Marine Biological Resources (RIMAR)</span>, Stazione Zoologica “Anton Dohrn”, 80121 Napoli, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BLuisa%2BChiusano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiusano%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BLuisa%2BChiusano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrea Cimarelli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Centre International de Recherche en Infectiologie (CIRI), Univ Lyon, Inserm, U1111, Université Claude Bernard Lyon 1, CNRS, UMR5308</span>, ENS de Lyon, 46 Allée d’Italie, 69007 Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BCimarelli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cimarelli%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BCimarelli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Francesco Conventi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>INFN sezione di Napoli, Complesso Universitario di Monte S. Angelo Edificio 6</span>, via Cintia, 80126 Napoli, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Dipartimento di Ingegneria Università degli studi di Napoli Parthenope, Centro Direzionale di Napoli</span>, Isola C 4, lato Sud, 80143 Napoli, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francesco%2BConventi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conventi%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrancesco%2BConventi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5575-1413" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Francesco Conventi</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonio Giannini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>University of Science and Technology of China (USTC)</span>, No.96, JinZhai Road, Baohe District, Hefei, Anhui, 230026, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BGiannini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giannini%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BGiannini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Stefan Hohenegger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institut de Physique des 2 Infinis (IP2I), CNRS/IN2P3, UMR5822</span>, 69622 Villeurbanne, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Université de Lyon, Université Claude Bernard Lyon 1</span>, 69001 Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BHohenegger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hohenegger%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BHohenegger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6564-0795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Hohenegger</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Francesco Sannino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>INFN sezione di Napoli, Complesso Universitario di Monte S. Angelo Edificio 6</span>, via Cintia, 80126 Napoli, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Dipartimento di Fisica E. Pancini, Università di Napoli Federico II, Complesso Universitario di Monte S. Angelo Edificio 6</span>, via Cintia, 80126 Napoli, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>Scuola Superiore Meridionale</span>, Largo S. Marcellino 10, 80138 Napoli, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>12</span><span class='nlm-institution'>CP3-Origins & the Danish Institute for Advanced Study, University of Southern Denmark</span>, Campusvej 55, DK-5230 Odense, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francesco%2BSannino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sannino%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrancesco%2BSannino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2361-5326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Francesco Sannino</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2021.07.22.21260952v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2021.07.22.21260952v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2021.07.22.21260952v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2021.07.22.21260952v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2021.07.22.21260952v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2021.07.22.21260952v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2021.07.22.21260952v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:425607" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:425607/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="biorxiv_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-biorxiv_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article abstract-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-4">Never before such a vast amount of data, including genome sequencing, has been collected for any viral pandemic than for the current case of COVID-19. This offers the possibility to trace the virus evolution and to assess the role mutations play in its spread within the population, in real time. To this end, we focused on the Spike protein for its central role in mediating viral outbreak and replication in host cells. Employing the Levenshtein distance on the Spike protein sequences, we designed a machine learning algorithm yielding a temporal clustering of the available dataset. From this, we were able to identify and define emerging persistent variants that are in agreement with known evidences. Our novel algorithm allowed us to define persistent variants as chains that remain stable over time and to highlight emerging variants of epidemiological interest as branching events that occur over time. Hence, we determined the relationship and temporal connection between variants of interest and the ensuing passage to dominance of the current variants of concern. Remarkably, the analysis and the relevant tools introduced in our work serve as an early warning for the emergence of new persistent variants once the associated cluster reaches 1% of the time-binned sequence data. We validated our approach and its effectiveness on the onset of the Alpha variant of concern. We further predict that the recently identified lineage AY.4.2 (‘Delta plus’) is causing a new emerging variant. Comparing our findings with the epidemiological data we demonstrated that each new wave is dominated by a new emerging variant, thus confirming the hypothesis of the existence of a strong correlation between the birth of variants and the pandemic multi-wave temporal pattern. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.</p><div id="sec-1" class="subsection"><p><strong>Highlights</strong></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-5"><strong>Objectives</strong> To study the relation among Spike protein mutations, the emergence of relevant variants and the multi-wave pattern of the COVID-19 pandemic.</p></li><li id="list-item-2"><p id="p-6"><strong>Setting</strong> Genomic sequencing of the SARS-CoV-2 Spike proteins in the UK nations (England, Scotland, Wales). Epi-demiological data for the number of infections in the UK nations, South Africa, California and India.</p></li><li id="list-item-3"><p id="p-7"><strong>Methodology</strong> We design a machine learning algorithm, based on the Levenshtein distance on the Spike protein sequences, that leads to a temporal clustering of the available dataset, from which we define emerging persistent variants. The above allows us to introduce the epidemiology of variants that we described via the Mutation epidemiological Renormalisation Group (MeRG) framework.</p></li><li id="list-item-4"><p id="p-8"><strong>Results</strong> We show that:</p><ol class="list-ord " id="list-2"><li id="list-item-5"><p id="p-9">Our approach, based only on the Spike protein sequence, allows to efficiently identify the variants of concern (VoCs) and of interest (VoIs), as well as other emerging variants occurring during the diffusion of the virus.</p></li><li id="list-item-6"><p id="p-10">Within our time-ordered chain analysis, a branching relation emerges, thus permitting to reconstruct the evolutionary diversification of Spike variants and the establishment of the epidemiologically relevant ones.</p></li><li id="list-item-7"><p id="p-11">Our analysis provides an early warning for the emergence of new persistent variants once its associated dominant Spike sequence reaches 1% of the time-binned sequence data. Validation on the onset of the Alpha VoC shows that our early warning is triggered 6 weeks before the WHO classification decision.</p></li><li id="list-item-8"><p id="p-12">Comparison with the epidemiological data demonstrates that each new wave is dominated by a new emerging variant, thus confirming the hypothesis that there is a strong correlation between the emergence of variants and the multi-wave temporal pattern depicting the viral spread.</p></li><li id="list-item-9"><p id="p-13">A theory of variant epidemiology is established, which describes the temporal evolution of the number of infected by different emerging variants via the MeRG approach. This is corroborated by empirical data.</p></li></ol></li></ul><ul class="list-unord " id="list-3"><li id="list-item-10"><p id="p-14"><strong>Conclusions</strong> Applying a ML approach to the temporal variability of the Spike protein sequence enables us to identify, classify and track emerging virus variants. Our analysis is unbiased, in the sense that it does not require any prior knowledge of the variant characteristics, and our results are validated by other informed methods that define variants based on the complete genome. Furthermore, correlating persistent variants of our approach to epidemiological data, we discover that each new wave of the COVID-19 pandemic is driven and dominated by a new emerging variant. Our results are therefore indispensable for further studies on the evolution of SARS-CoV-2 and the prediction of evolutionary patterns that determine current and future mutations of the Spike proteins, as well as their diversification and persistence during the viral spread. Moreover, our ML algorithm works as an efficient early warning system for the emergence of new persistent variants that may pose a threat of triggering a new wave of COVID-19. Capable of a timely identification of potential new epidemiological threats when the variant only represents 1% of the new sequences, our ML strategy is a crucial tool for decision makers to define short and long term strategies to curb future outbreaks. The same methodology can be applied to other viral diseases, influenza included, if sufficient sequencing data is available.</p></li></ul></div></div><h3>Competing Interest Statement</h3><p id="p-15">The authors have declared no competing interest.</p><h3>Funding Statement</h3><p id="p-16">None</p><h3>Author Declarations</h3><p id="p-17">I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.</p><p id="p-18">Yes</p><p id="p-19">The details of the IRB/oversight body that provided approval or exemption for the research described are given below:</p><p id="p-20">Not applicable.</p><p id="p-21">I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.</p><p id="p-22">Yes</p><p id="p-23">I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).</p><p id="p-24">Yes</p><p id="p-25">I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.</p><p id="p-26">Yes</p><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-others" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> <span class="em-link"><span class="em-addr">sannino{at}cp3.sdu.dk</span></span></p></li><li class="fn-update fn-group-summary-of-updates" id="fn-3"><p id="p-27">Extended analysis with early warning performance results and study of the spike protein diversification.</p></li></ul></div><div class="section data-availability" id="sec-12"><h2 class="">Data Availability</h2><p id="p-68">All raw data used in this work are obtained from open-source repositories: <a href="https://www.gisaid.org">https://www.gisaid.org</a> for the sequencing and <a href="https://ourworldindata.org/">https://ourworldindata.org/</a> for the epidemiological data. The Machine Learning code is available at <a href="https://github.com/AdeledeHoffer/ML-Covid">https://github.com/AdeledeHoffer/ML-Covid</a></p><p id="p-69">
<a href="https://github.com/AdeledeHoffer/ML-Covid">https://github.com/AdeledeHoffer/ML-Covid</a>
</p></div><span class="highwire-journal-article-marker-end"></span></div><span class="related-urls"></span></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-copyright" >
  
      
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-copyright field-type-text field-label-inline clearfix"><div class="field-label">Copyright&nbsp;</div><div class="field-items"><div class="field-item even">The copyright holder for this preprint is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.<span class="license-type"> It is made available under a <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="" data-icon-position="" data-hide-link-title="0">CC-BY-NC-ND 4.0 International license</a>.</span></div></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.07.22.21260952v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><span class="icon-chevron-up"></span> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2021.06.20.21259209v4" title="Vaccine Effectiveness for the COVID-19 in Japan" rel="prev" class="pager-link-prev link-icon"><span class="icon-circle-arrow-left"></span> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.07.25.20161984v2" title="Power Law in COVID-19 Cases in China" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <span class="icon-circle-arrow-right"></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 22, 2021.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2021.07.22.21260952v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><span class="icon-external-link-sign"></span> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2021.07.22.21260952v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2021/10/22/2021.07.22.21260952.external-links" target="_self" class="link-icon"><span class="icon-file"></span> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2021.07.22.21260952v2" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about medRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2021.07.22.21260952v2" class="active" data-icon-position="" data-hide-link-title="0">Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from medRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the medRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/425607" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XYRG9pjX_4FXeM4Yv2hKMI5RdXW4E92r_ZjK1lQNDJE" />
<input type="hidden" name="form_id" value="forward_form" />
<fieldset class="captcha form-wrapper"><legend><span class="fieldset-legend">CAPTCHA</span></legend><div class="fieldset-wrapper"><div class="fieldset-description">This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.</div><input type="hidden" name="captcha_sid" value="157693500" />
<input type="hidden" name="captcha_token" value="610f1c582d49ad27c882bfb1a2602fe5" />
<input type="hidden" name="captcha_response" value="Google no captcha" />
<div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image"></div></div></fieldset>
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><span class="icon-share-alt"></span> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node425607--3" data-node-nid="425607" id="node425607--41334398599" data-pisa="medrxiv;2021.07.22.21260952v2" data-pisa-master="medrxiv;2021.07.22.21260952" data-seqnum="425607" data-apath="/medrxiv/early/2021/10/22/2021.07.22.21260952.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Adele</span> <span class="nlm-surname">de Hoffer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Shahram</span> <span class="nlm-surname">Vatani</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Corentin</span> <span class="nlm-surname">Cot</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Giacomo</span> <span class="nlm-surname">Cacciapaglia</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Maria Luisa</span> <span class="nlm-surname">Chiusano</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Cimarelli</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Conventi</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Antonio</span> <span class="nlm-surname">Giannini</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Hohenegger</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Sannino</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">medRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2021.07.22.21260952; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2021.07.22.21260952 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;title=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;title=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;text=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;title=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;t=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;title=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;title=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><span class="icon-globe"></span> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article node425607--5" data-node-nid="425607" id="node425607--61383848445" data-pisa="medrxiv;2021.07.22.21260952v2" data-pisa-master="medrxiv;2021.07.22.21260952" data-seqnum="425607" data-apath="/medrxiv/early/2021/10/22/2021.07.22.21260952.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Variant-driven multi-wave pattern of COVID-19 via a Machine Learning analysis of spike protein mutations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Adele</span> <span class="nlm-surname">de Hoffer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Shahram</span> <span class="nlm-surname">Vatani</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Corentin</span> <span class="nlm-surname">Cot</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Giacomo</span> <span class="nlm-surname">Cacciapaglia</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Maria Luisa</span> <span class="nlm-surname">Chiusano</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Cimarelli</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Conventi</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Antonio</span> <span class="nlm-surname">Giannini</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Hohenegger</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Francesco</span> <span class="nlm-surname">Sannino</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">medRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2021.07.22.21260952; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2021.07.22.21260952 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations&amp;rft.volume&amp;rft.issue&amp;rft.date=2021-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=medRxiv&amp;rft.au=de+Hoffer%2C+Adele&amp;rft.au=Vatani%2C+Shahram&amp;rft.au=Cot%2C+Corentin&amp;rft.au=Cacciapaglia%2C+Giacomo&amp;rft.au=Chiusano%2C+Maria+Luisa&amp;rft.au=Cimarelli%2C+Andrea&amp;rft.au=Conventi%2C+Francesco&amp;rft.au=Giannini%2C+Antonio&amp;rft.au=Hohenegger%2C+Stefan&amp;rft.au=Sannino%2C+Francesco"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/425607/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/425607/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/425607/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/425607/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/425607/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/425607/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/425607/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/425607/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/425607/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/425607/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/425607/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/425607/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;count=horizontal&amp;via=&amp;text=Variant-driven%20multi-wave%20pattern%20of%20COVID-19%20via%20a%20Machine%20Learning%20analysis%20of%20spike%20protein%20mutations&amp;counturl=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.07.22.21260952v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.medrxiv.org/content/10.1101/2021.07.22.21260952v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/epidemiology" class="highlight" data-icon-position="" data-hide-link-title="0">Epidemiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">Addiction Medicine</a> <span class = "article-count">(111)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">Allergy and Immunology</a> <span class = "article-count">(299)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">Anesthesia</a> <span class = "article-count">(75)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">Cardiovascular Medicine</a> <span class = "article-count">(665)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">Dentistry and Oral Medicine</a> <span class = "article-count">(121)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">Dermatology</a> <span class = "article-count">(77)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">Emergency Medicine</a> <span class = "article-count">(216)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">Endocrinology (including Diabetes Mellitus and Metabolic Disease)</a> <span class = "article-count">(299)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(7268)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">Forensic Medicine</a> <span class = "article-count">(4)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">Gastroenterology</a> <span class = "article-count">(306)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">Genetic and Genomic Medicine</a> <span class = "article-count">(1295)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">Geriatric Medicine</a> <span class = "article-count">(126)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">Health Economics</a> <span class = "article-count">(308)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">Health Informatics</a> <span class = "article-count">(1004)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">Health Policy</a> <span class = "article-count">(526)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">Health Systems and Quality Improvement</a> <span class = "article-count">(359)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">Hematology</a> <span class = "article-count">(151)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">HIV/AIDS</a> <span class = "article-count">(287)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">Infectious Diseases (except HIV/AIDS)</a> <span class = "article-count">(8512)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">Intensive Care and Critical Care Medicine</a> <span class = "article-count">(483)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">Medical Education</a> <span class = "article-count">(160)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">Medical Ethics</a> <span class = "article-count">(40)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">Nephrology</a> <span class = "article-count">(153)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">Neurology</a> <span class = "article-count">(1239)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">Nursing</a> <span class = "article-count">(63)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">Nutrition</a> <span class = "article-count">(200)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">Obstetrics and Gynecology</a> <span class = "article-count">(255)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">Occupational and Environmental Health</a> <span class = "article-count">(368)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">Oncology</a> <span class = "article-count">(702)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">Ophthalmology</a> <span class = "article-count">(217)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">Orthopedics</a> <span class = "article-count">(70)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">Otolaryngology</a> <span class = "article-count">(139)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">Pain Medicine</a> <span class = "article-count">(73)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">Palliative Medicine</a> <span class = "article-count">(30)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(195)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">Pediatrics</a> <span class = "article-count">(410)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Therapeutics</a> <span class = "article-count">(195)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">Primary Care Research</a> <span class = "article-count">(158)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">Psychiatry and Clinical Psychology</a> <span class = "article-count">(1369)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">Public and Global Health</a> <span class = "article-count">(3057)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">Radiology and Imaging</a> <span class = "article-count">(509)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">Rehabilitation Medicine and Physical Therapy</a> <span class = "article-count">(244)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">Respiratory Medicine</a> <span class = "article-count">(415)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">Rheumatology</a> <span class = "article-count">(158)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">Sexual and Reproductive Health</a> <span class = "article-count">(118)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">Sports Medicine</a> <span class = "article-count">(115)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">Surgery</a> <span class = "article-count">(153)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">Toxicology</a> <span class = "article-count">(32)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">Transplantation</a> <span class = "article-count">(67)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">Urology</a> <span class = "article-count">(51)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__Vh5BcRYb6VtLoN3uam6O4DIKltYUVMjVDWtakoysPq0__Wed8jsPTirEozek7dWSCS8970Cp7a9xKKgFk6FPuSVM__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.js"></script>
<script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js"></script>
<script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__OWu9jWxbqz5eMqeTDc4xZhOzPlgoEqGh5PmxOqlf8ZY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
  </body>
</html>
